gk/fib.c

```diff
@@ -1867,7 +1876,7 @@ fillup_gk_neighbor_dump_entry(struct gk_neighbor_dump_entry *dentry,
 
 static void
 list_hash_table_neighbors(lua_State *l, enum gk_fib_action action,
```

Comment:
Rename `list_hash_table_neighbors()` to `list_hash_table_neighbors_unlock()` and move `rte_spinlock_unlock_tm(&ltbl->lock);` from its callers to the end of it. This change makes using this function a little less error prone.